test/test_jit.py

```diff
@@ -919,17 +921,19 @@ def get_forward_graph(m):
                    .check('ClassType<Observer> = prim::GetAttr[name="observer_for_') \
                    .check_next('prim::CallMethod[name="forward"](%observer_for_') \
                    .run(str(get_forward_graph(m._c)))
-        assert len([x for x, _ in m._c._get_module('conv')._get_modules()
-                    if x.startswith('observer_for_')]) == 3, \
-            'Expected to have 3 observer submodules'
-        FileCheck().check('ClassType<Observer> = prim::GetAttr[name="observer_for_') \
-                   .check_next('prim::CallMethod[name="forward"](%observer_for_') \
-                   .check('ClassType<Observer> = prim::GetAttr[name="observer_for_') \
-                   .check_next('prim::CallMethod[name="forward"](%observer_for_') \
-                   .check_next('Tensor = prim::CallMethod[name="conv2d_forward"](%self') \
-                   .check('ClassType<Observer> = prim::GetAttr[name="observer_for_') \
-                   .check_next('prim::CallMethod[name="forward"](%observer_for_') \
-                   .run(str(get_forward_graph(m._c._get_module("conv"))))
+        # we have duplicate observer for weight and output right now
```

Comment:
oh, it's because in forward function of Conv2d, we call `self.conv2d_forward`: https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/conv.py#L345, since now we also look into `conv2d_forward` and we didn't skip duplicated observers in this PR, we'll have some extra observer modules.